lib/drab/commander.ex

```diff
@@ -191,6 +208,11 @@ defmodule Drab.Commander do
   Runs after the event handler. Gets return value of the event handler function as a third argument.
   Can be filtered by `:only` or `:except` options, analogically to `before_handler`
 
+
+  ### `handle_info_message`
```

Comment:
This seems to focus a lot on 'pubsub' when it is nothing of the sort, only thing it does is pass in otherwise unhandled messages, I.E it's an info message handler, no need to focus on pubsub stuff and just say that when an unknown message is sent to the drab pid then it is received here.  Phoenix.PubSub already clarifies how Phoenix.PubSub works.

Reply:
Yes it is focused on PubSub because this is the main and most useful reason to add this stuff to Drab, but of course we change the documentation.